src/PrestaShopBundle/Twig/Component/NavBar.php

```diff
@@ -58,7 +59,7 @@ public function getPsVersion(): string
         return $this->psVersion;
     }
 
-    public function isCollapseMenu(): bool
+    public function isMenuCollapsed(): bool
```

Comment:
Isn't this one now a duplicate of the one from global variables? Maybe we can keep only one?

Reply:
Yes, we can use the global instant of this one